company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -137,7 +145,9 @@ public void sortedOrders()
     @Test
     public void whoOrderedSaucers()
     {
-        MutableList<Customer> customersWithSaucers = null;
+        Predicate<Order> orderPredicate = order -> order.getLineItems().stream().anyMatch(lineItem -> "saucer".equals(lineItem.getName()));
```

Comment:
Nikhil, instead of using `stream().anyMatch()` here in this predicate, use `anySatisfy`  as you are doing with `customersOrderedSaucers`. In order to use `anySatisfy()` here, you will need to modify `Order` class specifically the `lineItems`  data type to an `eclipse collections` one like MutableList, MutableSortedBag, etc.

Reply:
Yes, got it. I have updated the PR to use `anySatisfy` instead of `stream().anyMatch()`. Please review.

Thanks!